src/HealthChecks.AzureServiceBus/AzureServiceBusSubscriptionHealthCheck.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.ServiceBus.Management;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace HealthChecks.AzureServiceBus
+{
+    public class AzureServiceBusSubscriptionHealthCheck : IHealthCheck
+    {
+        private static readonly ConcurrentDictionary<string, ManagementClient> ManagementClientConnections =
+            new ConcurrentDictionary<string, ManagementClient>();
+        private readonly string _connectionString;
+        private readonly string _topicName;
+        private readonly string _subscriptionName;
+
```

Comment:
```suggestion
```

Reply:
I followed the convention in the TopicHealthCheck
https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/master/src/HealthChecks.AzureServiceBus/AzureServiceBusTopicHealthCheck.cs#L15

Reply:
What convention? I do not see.

Reply:
"private readonly string _connectionString;" from AzureServiceBusTopicHealthCheck
The member variable has _ prefix.